irksome/galerkin_stepper.py

```diff
@@ -201,9 +239,36 @@ def get_form_and_bcs(self, stages, basis_type=None, order=None, quadrature=None,
         return getFormGalerkin(F or self.F,
                                trial_el, test_el,
                                quadrature or self.quadrature,
-                               self.t, self.dt, self.u0, stages, self.orig_bcs)
+                               self.t, self.dt, self.u0, stages, self.orig_bcs,
+                               aux_indices or self.aux_indices)
 
     def _update(self):
         k1, = self.trial_el.entity_dofs()[0][1]
         for i, u0bit in enumerate(self.u0.subfunctions):
             u0bit.assign(self.stages.subfunctions[self.num_fields*(k1-1)+i])
+
+    def set_initial_guess(self):
```

Comment:
This logic could go away if we reformulate the stages to be an additive update to the current value of u0.

Reply:
Let's make that a separate issue since we need to think about bounds constraints in this context too.